include/envoy/network/dns.h

```diff
@@ -24,6 +24,8 @@ class ActiveDnsQuery {
   virtual void cancel() PURE;
 };
 
+enum class DnsLookupFamily { v4_only, v6_only, fallback };
```

Comment:
It seems there are enums with all caps (https://github.com/lyft/envoy/blob/998b8119ed57269e77d0a03b1b92e7a98c5d30d1/source/server/config/network/http_connection_manager.h#L114) but also many cases of not all caps (https://github.com/lyft/envoy/blob/998b8119ed57269e77d0a03b1b92e7a98c5d30d1/source/server/config/network/http_connection_manager.h#L24) Which is the convention?